cucumber/step_definitions/jobPage.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+const { expect } = require('chai');
+const CareerPage = require('../../pageObjects/careerPage');
+
+const careerPage = new CareerPage();
+
+
+const { Given, When, Then, setDefaultTimeout, After, Status } = require('cucumber');
+const { Button, browser } = require('protractor');
+const fs = require('fs');
+const path = require('path');
+
+

```

Comment:
Do you think some empty lines can be removed from this file?

Reply:
Also I noticed this in other files, let's consider this a general suggestion then.